acceptance/bundle/deploy/secret-scope/script

```diff
@@ -0,0 +1,15 @@
+envsubst < databricks.yml.tmpl > databricks.yml
+
+cleanup() {
+    trace $CLI bundle destroy --auto-approve
+}
+trap cleanup EXIT
+
+trace $CLI bundle deploy
+trace $CLI bundle summary --output json | jq '.resources.secret_scopes.secret_scope1'
+
+trace $CLI secrets list-scopes -o json | jq '.[] | select(.name == "my-secrets")'
+trace $CLI secrets list-acls my-secrets
```

Comment:
Does the test configure ACLs?

Reply:
it now does (added permissions in a follow-up commit)